File: {{cookiecutter.project_slug}}/code/{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}.py
"""
Main module for {{ cookiecutter.project_slug }}

Author:  {{ cookiecutter.full_name }}
Contact: {{ cookiecutter.email }}
Years:   {% now 'local', '%Y' %}
"""

# %% Import
from {{ cookiecutter.project_slug }}.configs import path_to, params
from {{ cookiecutter.project_slug }}.preprocessing.freesurfer import foo

# %% Set global vars & paths  >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o
pass

# %% Functions  >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o

def main():
    foo()
    print(f"{path_to.results.GLM}/{params.weight_decay}/")

# %% __main__ o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o

if __name__ == "__main__":
    # Run main
    main()

#  o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o >><< o END

File: research-project-comprehensive/{{cookiecutter.project_slug}}/code/Rscripts/{{cookiecutter.project_slug}}.R
# Title: {{ cookiecutter.project_slug }}
# Description:  This script is an example script using R in your research project.
#               Adapt it to your liking.
# Author: {{ cookiecutter.full_name }}
# Years: {% now 'local', '%Y' %}

# package dependencies
library(ggplot2)
source("code/Rscripts/configs.R")
# exposes config, paths, params, PROJECT_NAME, PROJECT_ROOT

# session info
# setwd(PROJECT_ROOT)
print(paste("Current working dir:", as.character(getwd())))
sessionInfo()
print(paste("The data folder is at:", config$paths$DATA))
print(paste("The weight decay is set to:", config$params$weight_decay))

# code
print("put your R code here")
